app/views/courses/CoursesView.coffee

```diff
@@ -119,6 +120,25 @@ module.exports = class CoursesView extends RootView
         @render()
       @supermodel.trackRequest(levels.fetchForClassroom(classroomID, { data: { project: "original,primerLanguage,slug,i18n.#{me.get('preferredLanguage', true)}" }}))
 
+    if features.china and @classrooms.find {id: '5d0082964ebb960059fc40b2'}
+      if new Date() >= new Date(2019, 5, 19, 12) && new Date() <= new Date(2019, 5, 25, 0)
+        if window.serverConfig?.currentTournament or true
```

Comment:
This seems to be always true, was this (`or true`) left from debugging?

Reply:
Missed that, thanks! Removed.